core/Plugin/Archiver.php

```diff
@@ -91,6 +181,35 @@ final public function callAggregateDayReport()
         try {
             ErrorHandler::pushFatalErrorBreadcrumb(static::class);
 
+            $pluginName = $this->getPluginName();
+
+            if (!empty($pluginName)) {
```

Comment:
With the comment within the `getPluginName` method and with the `null` value removed, this could be
```suggestion
            if ($this->isPluginLoaded()) {
```